src/Filament/Pages/Dashboard.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Taecontrol\MoonGuard\Filament\Pages;
+
+use Filament\Panel;
+use Filament\Pages\Page;
+use Filament\Facades\Filament;
+use Illuminate\Support\Facades\Route;
+use Filament\Support\Facades\FilamentIcon;
+use Illuminate\Contracts\Support\Htmlable;
+
+class Dashboard extends Page
+{
+    protected static ?int $navigationSort = -2;
+
```

Comment:
Why the navigation sorting is a negative number?

Reply:
Honestly, this is the default value from Filament, I haven't into it.